src/components/Marketplace/FarmCard.js

```diff
@@ -0,0 +1,228 @@
+    isDelivery,
+  } = props;
+
+  // Set capabilityTags (PACA, cold chain, delivery)
+  const capabilityTags = [];
+  if (isPACA === true) {
+    capabilityTags.push('PACA Certified');
+  }
+  if (isColdChain === true) {
+    capabilityTags.push('Cold Chain Capabilities');
+  }
+  if (isDelivery === true) {
+    capabilityTags.push('Delivery');
+  }
+
+  return (
+    <Card
+      className={[
+        styles.cardContainer,
+        isExpanded ? styles.expandedBorder : styles.collapsedBorder,
+      ]}
+      variant="outlined"
+    >
+      {/* Header/non-collapsed card contains farm name, expand/collapse icon, tags for op type */}
+      <Grid
+        container
+        direction="row"
+        justify="space-between"
+        alignItems="center"
+        className={styles.headerContainer}
+      >
+        {/* Contains title (farm name) and subtitle (location) */}
+        <Grid
+          container
+          direction="row"
+          justify="space-between"
+          alignItems="start"
+          className={styles.titleContainer}
+        >
+          <div className={styles.titleText}>
+            {farmName}
+          </div>
+          <div className={styles.subTitleText}>
+            {address}
+          </div>
+          <div className={styles.subTitleText}>
+            {zipCode}
+          </div>
+        </Grid>
+        <IconButton
+          className={styles.iconColour}
+          // On icon click, collapse or expand the card
+          onClick={() => setIsExpanded(!isExpanded)}
+        >
+          {!isExpanded && <ExpandMoreIcon />}
+          {isExpanded && <ExpandLessIcon />}
+        </IconButton>
+      </Grid>
+      <CardContent className={styles.collapsedContent}>
+        {operationTypeTags.map((tag) => (
+          <Chip
+            label={tag}
+            className={styles.operationTags}
+            size="medium"
+            variant="default"
+            classes="tags"
+          />
+        ))}
+      </CardContent>
+      {/* If uncollapsed, shows description and additional tags */}
+      <Collapse in={isExpanded} timout="auto" unmountOnExit>
+        <CardContent className={styles.expandedContent}>
+          <div>
+            <div className={styles.descriptionContainer}>
+              {description}
+            </div>
+            {capabilityTags.map((tag) => (
+              <Chip
+                label={tag}
+                className={styles.capabilityTags}
+                size="small"
+                variant="default"
+                icon={<CheckCircleOutlineIcon className={styles.iconColour} />}
+              />
+            ))}
+          </div>
+          <Grid /* Used to keep Market + market tags in 1 row */
+            container
+            direction="row"
+            alignItems="center"
+            className={styles.marketContainer}
+          >
+            <div>
+              Market:
+            </div>
+            {marketTags.map((tag) => (
```

Comment:
Since we're gonna be deleting marketTags, you can replace your usage of marketTags to display the tags of FarmingType! Just make sure to change the color to modify it to match the FIgma. Also, it's okay if it's  on the left hand side because we're removing the check marks anyway